salt/metalk8s/orchestrate/upgrade/post.sls

```diff
@@ -2,3 +2,4 @@
 
 include:
   - metalk8s.addons.prometheus-operator.post-upgrade
+  - metalk8s.kubernetes.cni.loopback.absent
```

Comment:
No you cannot include it as is here, this state is run on the salt-master so if you do this it will just remove the cni in the salt master container (so likely do not remove anything)
So to me you need to create another orchestrate that run this on every minions
OR
(maybe better) just include this absent in the calico state with a comment to say we can remove it in 2.7. Since we do not need this one once calico is configured as CNI

Reply:
Or maybe we want to remove all others CNI plugins when we deploy calico :thinking: I don't know if it can cause trouble to have different CNI plugins
@NicolasT @gdemonet Any idea ?
(Anyway it's not the goal of this PR)

Reply:
Shouldn't cause 'trouble' since they're picked based on filename, hence the `10-` and `30-` (or something) prefix we use.

Reply:
Yeah, I saw during my test that the file was not properly removed, then I forgot to change it.
It's now done.